tests/test_cli.py

```diff
@@ -64,6 +64,25 @@ def test_cli_dspace_mets(caplog, runner, tmp_path):
     assert "Completed transform, total record count: 12" in caplog.text
 
 
+def test_cli_whoas(caplog, monkeypatch, runner, tmp_path):
```

Comment:
Can remove unused `monkeypatch` arg. Honestly I'm surprised (and mildly annoyed) that our linters don't catch that!

Reply:
It does show up as faded in VS Code so I should have caught it and I'll keep on eye out going forward